Source/SPNetworkPreferencePane.m

```diff
@@ -178,8 +179,8 @@ - (IBAction)pickSSHClientViaFileBrowser:(id)sender
 			forKeyPath:SPHiddenKeyFileVisibilityKey
 			   options:NSKeyValueObservingOptionNew
 			   context:NULL];
-	
-	[_currentFilePanel beginSheetModalForWindow:[_currentAlert window] completionHandler:^(NSInteger result) {
+
+	[_currentFilePanel beginSheetModalForWindow:[[NSApplication sharedApplication] keyWindow] completionHandler:^(NSInteger result) {
```

Comment:
I believe keyWindow is deprecated? Might be wrong

Reply:
<img width="494" alt="Screen Shot 2020-07-09 at 0 16 21" src="https://user-images.githubusercontent.com/7204168/86975811-6e107f80-c179-11ea-8e9e-6ce25fb20a6f.png">

Doesn't look like, no warning.

Reply:
Ahhh only deprecated for iOS and catalyst apps
https://stackoverflow.com/questions/57134259/how-to-resolve-keywindow-was-deprecated-in-ios-13-0